docs/storage/prometheus.md

```diff
@@ -14,79 +14,79 @@ To monitor cAdvisor with Prometheus, simply configure one or more jobs in Promet
 
 The table below lists the Prometheus container metrics exposed by cAdvisor (in alphabetical order by metric name):
 
-Metric name | Type | Description | Unit (where applicable)
```

Comment:
I think that it will be good to indicate when special build of cAdvisor is needed e.g. with libpfm4 or libimpctl

Reply:
Good point but maybe we need another PR?